src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -71,7 +73,7 @@ class MarathonScheduler @Inject() (
   override def statusUpdate(driver: SchedulerDriver, status: TaskStatus): Unit = {
     log.info("Received status update for task %s: %s (%s)"
       .format(status.getTaskId.getValue, status.getState, status.getMessage))
-
+    healthCheckManager.reconcile(status)
```

Comment:
@meichstedt: Instead of hard wiring the reconcile logic here: can we make this part of `NotifyHealthCheckManagerStepImpl`?